### PR TITLE
you-roz Restaurant

### PR DESCRIPTION
**Pull request by zakiyousufi**
computer

### Attention Please
**PR[#3](https://github.com/Roseokpe/E-commerce/pull/31)** 
This PR is for adding all necessary files to the project and following estlint standards. this is a one-page website that contains Food lists that get from API, you can like the food,  add your comment, and reserve a table for your self and the API will post your information.

Me @zakiyousufi and my teammate @Roseokpe  have followed the project requirements, Github flow, and git flow 
✅ All linters checks are passing. 
**_Our project includes >>>>>_**
`index.html` for creating the Header and Footer and project structure
 `index.js`  our main codes are in this files

- `popup.js` Includes the all information about food and comments
- `reservation.js` Includes the all information about reservations and new reservation
- `home.js` Includes the all information about the home page
We have  --mock-- folder that includes `popuo.js`, `reservations.js`, `home.js` that have counters for Items, Comments, and reservations.
We have created 3 test files for testing the counters
`meales.test.js`, `reservation.test.js`, and `popup.test.js`,

 `style.css` for styling the project

Have fun 😊😉

### CODE REVIEW

File: __mocks__/popup.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+const userComment = [
+  {
+    item_id: '1',
+    username: 'Maria',
+    comment: 'Yummy',
+  },
+];
+
+export const displayPopup = () => {
+  comments.push(userComment);
```
Comment (topeogunleye) [inline __mocks__/popup.js]: - [ ] As written in the project requirement. Please kindly add a pop up window for your user comments. Your pop up window does not seem to work when I click on the comments. 
- [ ] Kindly ensure you use a pop up modal and do not append the data to the bottom of the page.

File: README.md [hunk 2]
```diff
@@ -1,73 +1,81 @@
...
 
+After the selecting location
 
-## Authors
+ >Type git clone in the terminal, paste the URL you copied earlier, and press “enter” to create your local clone.
 
-👤 **Author1**
+ >Syntax:
+  >>> cd [folder name] eg: desktop 
+  >>> git clone [repository URL]
```
Comment (topeogunleye) [inline README.md:62]: - [ ] As written in the project requirements, I kindly suggest that you add a video demo of your project presenting the features of your project. The video should consist of the following:

- The maximum length of the video is 5 minutes.
- The video contains a clear and objective description about the project in professional English. The topic and purpose of the project are made clear.
- All the members in the video should appear in the video and talk about the project.
- The video contains a highlight about a piece of the code the student finds interesting or is proud of making.

File: src/module/home.js [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+            card.innerHTML = `${like.likes}`
+          }
+        })
+      }
+    })
+  }
+ 
+}
+
+const display = async () => {
```
Comment (topeogunleye) [inline src/module/home.js]: - [ ] I kindly suggest that you follow the given mockup to create your home page and also add the items counter on the Home Page.
- [ ] I kindly suggest that you implement the pop up and don't append the data to the bottom of the page.

File: README.md [hunk 4]
```diff
@@ -1,73 +1,81 @@
...
+- following linters standards.
+- following github flows standards.
+- following git flows standards.
 
-## Getting Started
 
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+### 🖥️LIVE DEMO
+You can use Leaderboard by clicking [here]()
```
Comment (topeogunleye) [inline README.md]: - [ ] I kindly suggest that you add your live demo link here.

**Review by topeogunleye — CHANGES_REQUESTED** [review]
Hi Team, 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!

> ## (Highlights) Good Points: 
> * No linter errors. :heavy_check_mark:
> * Followed Github flow :heavy_check_mark:
> * PR has a good title and summary :heavy_check_mark:

## Suggested changes
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Nothing to mention

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

> Feel free to leave any questions or comments in the PR thread if something is not 100% clear
> Please, remember to tag @topeogunleye in your question so I can receive the notification.
>
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by Kandy-Peter — APPROVED** [review]
Hi @zakiyousufi and @Roseokpe 

### APPROVED 🟢 

Great implementation of all the requirements in this milestone. 👏
Your web app looks really fantastic. 🎉 
image
_You have done an amazing job 👍_

Your project is complete!  There is nothing else to say other than... it's time to merge it. :shipit: 

##  Optional suggestion

 
# CHEERS AND HAPPY CODING!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**